drivers/SmartThings/zigbee-motion-sensor/src/aqara/init.lua

```diff
@@ -14,29 +28,19 @@ local is_aqara_products = function(opts, driver, device)
   return false
 end
 
-local function detection_frequency_capability_handler(driver, device, command)
-  local frequency = command.args.frequency
-  aqara_utils.set_pref_changed_field(device, aqara_utils.PREF_FREQUENCY_KEY, frequency)
-  aqara_utils.write_custom_attribute(device, aqara_utils.FREQUENCY_ATTRIBUTE_ID, frequency)
-end
+local function device_init(driver, device)
+  battery_defaults.build_linear_voltage_init(2.6, 3.0)(driver, device)
 
-local function detection_frequency_attr_handler(driver, device, value, zb_rx)
-  aqara_utils.set_detection_frequency(device, value.value)
+  for _, attribute in ipairs(CONFIGURATIONS) do
+    device:add_configured_attribute(attribute)
+    device:add_monitored_attribute(attribute)
+  end
 end
 
 local aqara_motion_handler = {
   NAME = "Aqara Motion Handler",
-  capability_handlers = {
-    [aqara_utils.detectionFrequencyId] = {
-      [aqara_utils.detectionFrequencyCommand] = detection_frequency_capability_handler,
-    }
```

Comment:
the sub-drivers already would have inherited this behavior. There is no need to duplicate code in two sub-drivers of the same parent driver. Just include the code in the parent driver.

Reply:
I updated.